JustSaying/AwsTools/MessageHandling/SqsNotificationListener.cs

```diff
@@ -143,54 +135,52 @@ private static string AggregateExceptionDetails(AggregateException ex)
             return innerExDetails.ToString();
         }
 
-        public void StopListening()
-        {
-            _cts.Cancel();
-            _log.LogInformation($"Stopping Listening - Queue: {_queue.QueueName}, Region: {_queue.Region.SystemName}");
-        }
-
```

Comment:
👍

Reply:
Is it it possible to keep the ` _log.LogInformation($"Stopping Listening...` message in some form? 
It can be useful in determining if a worker shut down gracefully or not.
Maybe just put it after the end of the `while (!ct.IsCancellationRequested) {  }` loop?

Reply:
`LogTaskEndState` will log a very similar message which includes queue name and region